### PR TITLE
docs: update documentatation

### PR DESCRIPTION
**Pull request by jensmo98**
Updated documentation in md file for storefront. Can and should be copied to README. 

1. Complimented with more documentation about how to construct the JSON for org chart.
2. Updated old documentation

### CODE REVIEW

**Review by Barsnes — CHANGES_REQUESTED** [review]
Overall good additions, left some suggestions, some for spellingmistakes, and a few questions about column counts.

File: apps/storefront/app/routes/dokumentasjon.mdx [hunk 1]
```diff
@@ -106,10 +106,12 @@ To ensure the chart functions correctly, it is designed to adhere to a set of ru
 - A column can contain a node with children, or two nodes that share children.
 - A column containing two nodes that share children will take up twice as much space as a column with only one node. We refer to this type of column as a special column.
 - A node that is to have children can have as many children as the user desires.
+- The first column must be a regular column, and should be aligned or have children.
+- If not the column contains the root node, and is one of maximum two columns in the row, you have the ability to align it.
```
[the remark this answers is not in the document]
Comment (jensmo98) [reply apps/storefront/app/routes/dokumentasjon.mdx:110]: Better?

[the remark this answers is not in the document]
Comment (Barsnes) [reply apps/storefront/app/routes/dokumentasjon.mdx]: Yes, looks great!

**Review by Barsnes — APPROVED** [review]
Looks great, good work 🐑

**Pull request merged**